Lib/fontbakery/profiles/fvar.py

```diff
@@ -454,35 +455,30 @@ def com_adobe_fonts_check_varfont_valid_default_instance_nameids(ttFont,
         if inst_coords == axes_dflt_coords:
             subfam_nameid = inst.subfamilyNameID
             postscript_nameid = inst.postscriptNameID
+            postscript_name = name_table.getDebugName(postscript_nameid) or "None"
 
             # Special handle the 0xFFFF case, to avoid displaying the value as 65535
             if postscript_nameid == 0xFFFF:
                 postscript_nameid = "0xFFFF"
 
-            inst_name = None
-            if name_table is not None:
-                inst_name = name_table.getDebugName(subfam_nameid)
+            inst_name = name_table.getDebugName(subfam_nameid) or f"Instance #{i}"
 
-            if inst_name is None:
-                inst_name = f"Instance #{i}"
-
-            if subfam_nameid not in {2, 17}:
+            if subfam_nameid not in {2, 17} and inst_name != name2:
```

Comment:
The rationale for this check says that the name of the default instance should be ID 2 or 17 or something with the same value, but here it checks only if the value is the same as name ID 2.

So the check will pass if the name of the default instance is ID 17, but fail if it is another name ID with the same value as ID 17.

Reply:
I'll have a look. Thanks for reporting it.

Reply:
@jenskutilek, when reporting problems on a merged PR (or on a closed issue), please consider doing so by opening a new issue so that there's no risk of the report being lost / forgotten.

Reply:
@jenskutilek are you using the latest pre-release? I believe the problem was fixed by #3896

Reply:
Indeed it is already fixed. Sorry, I didn’t see that. Using the release version.